### PR TITLE
No longer inject MathJax config by default

### PR DESCRIPTION
**Pull request by archmoj**
cc: #5395 

@plotly/plotly_js

### CODE REVIEW

File: src/lib/svg_text_utils.js [hunk 1]
```diff
@@ -153,27 +153,16 @@ function cleanEscapesForTex(s) {
...
             MathJax.Hub.processSectionDelay = 0;
         }
-
-        return MathJax.Hub.Config({
-            messageStyle: 'none',
-            tex2jax: {
-                inlineMath: [['$', '$'], ['\\(', '\\)']]
-            },
-            displayAlign: 'left',
-        });
```
Comment (alexcjohnson) [inline src/lib/svg_text_utils.js:176]: I think we still want all of the code you removed from `svg_text_utils` - store the existing config, set our own, and replace the original config at the end. That way we get the config we want without interfering with anything the rest of the page might be doing.

**Review by alexcjohnson — APPROVED** [review]
💃 Love it.

**Comment by nicolaskruchten** [comment]
@alexcjohnson this PR is the one that makes `window.PlotlyConfig = {MathJaxConfig: 'local'}` the default behaviour, correct?

**Comment by alexcjohnson** [comment]
Yes. `window.PlotlyConfig` is no longer a thing after this PR, and the sole behavior is what `window.PlotlyConfig = {MathJaxConfig: 'local'}` did previously.

**Pull request merged**